test

```diff
@@ -1,5 +1,11 @@
 #!/bin/bash
 
+if [ -v TRAVIS_PULL_REQUEST_BRANCH ]; then
```

Comment:
This look more logic

Reply:
Yes, but seems that `-v` is not sufficient here because is set somewhere as empty. But if I use `-z` instead this can be fixed

Reply:
I think we need ! -z because -z means is empty

Reply:
! -z just skipped both builds ..